pandas/tests/tools/test_to_datetime.py

```diff
@@ -2443,3 +2443,23 @@ def test_na_to_datetime(nulls_fixture, klass):
     result = pd.to_datetime(klass([nulls_fixture]))
 
     assert result[0] is pd.NaT
+
+
+def test_empty_string_datetime_coerce():
+    # GH13044
+    td = Series(["May 04", "Jun 02", ""])
+    format = "%b %y"
+
+    # coerce empty string to pd.NaT
+    result = pd.to_datetime(td, format=format, errors="coerce")
+    expected = Series(["2004-05-01", "2002-06-01", pd.NaT], dtype="datetime64[ns]")
+    pd.testing.assert_series_equal(expected, result)
+
+    # raise an exception in case a format is given
+    with pytest.raises(ValueError, match="does not match format"):
+        result = pd.to_datetime(td, format=format, errors="raise")
+
+    # don't raise an expection in case no format is given
+    result = pd.to_datetime([1, ""], unit="s", errors="raise")
```

Comment:
can you make this *excatly* the same as L2454 but w/o the coerce (e.g. unit is a completely different path)

Reply:
I just changed the input data slightly, because the previous series cause the following error:
```
OutOfBoundsDatetime: Out of bounds nanosecond timestamp: 1-05-04 00:00:00
````
when not specifying any format.

Reply:
ahh ok